spec/paperclip/attachment_registry_spec.rb

```diff
@@ -111,6 +111,34 @@
 
       assert_equal expected_definitions, definitions
     end
+
+    it 'allows subclasses to override attachment defitions' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.